spaceship/spec/provisioning_profile_spec.rb

```
@@ -54,6 +54,30 @@
     end
   end
 
+  describe '#all via xcode api' do
+    around(:all) do |example|
+      switch = ENV['SPACESHIP_AVOID_XCODE_API']
+      example.run
```

if for some reason the `example.run` fails hard.